### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by marvinsxtr** [covidpass-org/covidpass#21]
Localize the web app for all EU countries.

**Comment by Ein-Tim** [comment]
I would like to help with the localization. 
So if you prepare the codebase, feel free to ping me here and I can take over the German translation (and can also take a look at the French one - no promise I can translate everything in high quality there).

**Comment by hauketoenjes** [comment]
Hi, thanks for your offer 👍🏼 

We will prepare the codebase while we transition to typescript (Issue #20). We will leave this open and inform you as soon as we did that.

**Comment by marvinsxtr** [comment]
@Ein-Tim The [localization branch](https://github.com/covidpass-org/covidpass/tree/localization) is now ready for the German translation.

**Comment by Ein-Tim** [comment]
@marvinsxtr 

Thanks for the update, I'll take a look at it later and will probably start with the translation tomorrow/on Sunday. 

If I need any advice or help, I can surely ping you here, right?

**Comment by marvinsxtr** [comment]
Of course!

**Comment by Ein-Tim** [comment]
Okay, so I took a look at the codebase, I assume I'll do the following: 
Create a new folder "locales/de" [here](https://github.com/covidpass-org/covidpass/tree/localization/public), copy the files from "locales/en" there and then just translate.

Do I have to translate any other files (which are providing the strings for the actual wallet card?

**Comment by marvinsxtr** [comment]
That's right, you can also add "de" to the locales list in `next-i18next.config.js`. We will handle translation of the wallet card on the server, that's why it is not included.

**Comment by Ein-Tim** [comment]
@marvinsxtr 

I now added the first translation to the files `common.yml`, `errors.yml` & `index.yml` with https://github.com/Ein-Tim/covidpass/commit/cbef4bea76d758b55b19e49ff9df265db295c3df. 
I did not (yet) translate the `imprint.yml` & `privacy.yml` files, since these are legal texts and I assume you would like to do this (also the Imprint seems to have been translated from German to English already, and if I now would translate English to German again, I assume this sounds a bit odd).

Not sure how to modify `next-i18next.config.js`, I would like to leave this to you. 

Should I provide a PR with my current changes or is there anything else left to do?

**Comment by marvinsxtr** [comment]
Thanks for the contribution, you are right with the legal stuff, that would be odd. Feel free to open a PR.

**Comment by Ein-Tim** [comment]
PR #26 is open for this. 

A little site note, the text is currently "Add your EU Digital Covid **Vaccination** Certificates to your favorite wallet app.", maybe we should remove "Vaccination" since this tool also supports recovered and tested certificates, or?

**Comment by marvinsxtr** [comment]
It doesn't yet but we will remove that as soon as we support these certificates.

**Comment by j3r3m067** [comment]
I can translate in French if you want ;)

**Comment by marvinsxtr** [comment]
@j3r3m067 That would be great! Would you also do the privacy policy and imprint? The latter is less important but would be nice for consistency. Otherwise just copy the english translation :)

**Comment by marvinsxtr** [comment]
Dutch and Finnish translations are now finished. See #48.

**Comment by everdrone** [comment]
Hi, love the project
I can translate in Italian

**Comment by marvinsxtr** [comment]
@everdrone Great! Feel free to open a PR :)

**Comment by vebaev** [comment]
I can help with Bulgarian if needed.

**Comment by marvinsxtr** [comment]
@vebaev The goal is to translate it for all EU countries, so I would appreciate it :)

**Comment by mcvendrell** [comment]
Hi guys, I'm ready for spanish translation.
It is just to translate the `locale/en` to `locale/es` and add to `next-i18next.config.js`? Nothing more?

**Comment by mackonsti** [comment]
Hi @marvinsxtr thanks for adding the colour palette to the main page, it's nicer and less complex for translating colours! Are you going to remove the translation keys for the colours then, or should we do this, in some next update? Many thanks!

**Comment by marvinsxtr** [comment]
@mackonsti I was thinking about using them for alt texts so colorblind people can use the web app too but for that, we would have to translate the new colors too. Apart from that the extra translation keys do no harm i guess.

**Comment by unk1nd** [comment]
have created a pull-request regarding norwegian translation

**Comment by nilaallj** [comment]
PR sent for Swedish translation! :)

**Comment by unk1nd** [comment]
https://github.com/covidpass-org/covidpass/pull/110

Why didn't I get Contributor tag for this :/

**Comment by hauketoenjes** [comment]
![image](https://user-images.githubusercontent.com/25830076/144932943-e3864a6e-c9e4-4a50-81af-b922fbfa272e.png)

Your commits show in the commit list, but for some reason, GitHub didn't connect them to your account…

**Comment by unk1nd** [comment]
Did I commit with wrong author config?

**Comment by marvinsxtr** [comment]
@unk1nd Your profile picture is now visible in the commit history for me so that seems to work.

**Comment by mackonsti** [comment]
Hello everyone, Season Greetings!

@marvinsxtr thanks again for your continuing updates!
I just reused the website for a new pass and saw the addition of the new string in `index.yml`.
Is there a way to notify all translators of new additions so we can add them together at once? (and be notified)

Please, can you kindly change/fix this word that Google Translate killed? 😅 
In file `/public/locales/el/index.yml`
The last new line says `share: Συστήσει`

It actually is not translated correctly, should be:
`share: Κοινοποιήστε το`

Thank you for pushing this change ASAP.
Happy new year to you and everyone and your loved ones.

**Comment by marvinsxtr** [comment]
@mackonsti Fixed it with #144.

**Comment by KoNickss** [comment]
I can offer to localise it to Romanian, how does this work? do I just open a pull request? where are the localised pages stored?

**Comment by marvinsxtr** [comment]
@KoNickss Thank you for your offer. You can create a new folder  with your translations in public/locales. There also need to be symlinks and some changes in a config file but I can do that part. Please open a pull request to the localization branch from the dev branch (the latest changes were not merged into localization yet).

**Comment by KoNickss** [comment]
Ok, I'll do so when I catch some time, thank you for the help and for this awesome project!

**Comment by KoNickss** [comment]
@marvinsxtr almost done, do I put diacritics or not? Commonly they aren't used when typing in romanian for convenience (since you can tell most words out without them) but they are used in academic and/or professional backgrounds, so do I put them in or not?

**Comment by marvinsxtr** [comment]
@KoNickss Just do what you find more suiting.

**Comment by KoNickss** [comment]
@marvinsxtr I am so sorry for bothering you so much and I truly apologise, but I've encountered somewhat of a roadblock, in the privacy file I need to define "the", while in romanian "the" isn't a word, it's a suffix (ul/o) you add to words to make them articulate (Certificate = Certificat; The certificate = Certificat_ul_), so how should I account for this?

**Comment by KoNickss** [comment]
If 'the' is only applied to phrases like "covid certificate", "privacy policy" or "schema" these are already articulated, so I'll leave the 'the' empty, if any grammar problems arrise I'll sort them out but I doubt there will be any

**Comment by marvinsxtr** [comment]
@KoNickss Sounds good. No worries. I don't think the index.yml on the localization branch already has the key share: in it, would be nice if you could add it with the translation as well.
